File: RentAppProject/Rent.Shared/Models/MoreWork.cs
﻿using System;
namespace Rent.Shared.Models
{
    public class MoreWork
    {
        public int ID { get; set; }

        //person to complete the task
        public int UserID { get; set; }
        public int LocationID { get; set; }
        public float? Hours { get; set; }

        //user who ordered the work
        public int CreatedByUserID { get; set; }

        public string Description { get; set; }
        public int CleaningPlanID { get; set; }
        public string CleaningPlanDescription { get; set; }
        public DateTime ExpectedCompletedTime { get; set; }
        public DateTime? ActualCompletedTime { get; set; }
    }
}

File: rent_dashboard/lib/src/models/morework.dart
class MoreWork {
  final int cleaningPlanID;
  final String description;
  final DateTime expectedTime;
  MoreWork.fromJson(json)
      : cleaningPlanID = json['cleaningPlanID'],
        description = json['description'],
        expectedTime = DateTime.parse(json['expectedCompletedTime']);
}
